packages/babel-preset-default/index.js

```diff
@@ -1,36 +1,74 @@
 module.exports = function( api ) {
+	let wpBuildOpts = {};
+	const isWPBuild = ( name ) => {
+		const result = [ 'WP_BUILD_MAIN', 'WP_BUILD_MODULE' ].some(
```

Comment:
`result` variable can be inlined, the way it was before 😄

Reply:
Blended it together with other things at e7dc7f1